acceptance/acceptance_test.go

```diff
@@ -38,6 +54,18 @@ var Scripts = map[string]bool{
 }
 
 func TestAccept(t *testing.T) {
+	testAccept(t, InprocessMode, "")
+}
+
+func TestInprocessMode(t *testing.T) {
+	if InprocessMode {
+		t.Skip("Already tested by TestAccept")
+	}
+	require.NotEmpty(t, testAccept(t, true, "help"))
```

Comment:
Not clear what "not empty" means here.

Reading along, I see that `testAccept` returns the # of cases that were run. To make it clear this could:
```go
numTestCases := testAccept(t, true, "help")
require.NotZero(t, numTestCases)
```

Reply:
Changed to NonZero.